doxygen/mbedtls.doxyfile

```diff
@@ -27,3 +27,15 @@ HAVE_DOT               = YES
 DOT_GRAPH_MAX_NODES    = 200
 MAX_DOT_GRAPH_DEPTH    = 1000
 DOT_TRANSPARENT        = YES
+
+# Doxygen accepts empty descriptions for commands such as \retval,
+# but clang -Wdocumentation doesn't (since Clang 15, for \retval).
+#   https://github.com/Mbed-TLS/mbedtls/issues/6960
+#   https://github.com/llvm/llvm-project/issues/60315
+# As a workaround, when documenting the status codes that a function can
+# return, if you don't have anything to say beyond the status code's
+# description, you can write something like
```

Comment:
Surely this should be "if you don't have anything to say beyond the name of the status code" (rather than `description`, since we are specifically saying it's empty)

(minor comment: I personally prefer `nodescription`, but if you think `empty` is better that's fine)

Reply:
What I wrote is what I meant, but I can see it's not clear, so I'll reformulate.

Ideally, for each status code that a function can return, we would document, in English, the circumstances under which the function returns this status. But in many cases, we'd mostly be repeating the documentation of the status code. That's not just the name of the status code, but what you can read by following the link to the status code constant.

There is always a description, it's just empty sometimes. So “empty description” is technically correct and “no description” isn't. I don't see how “no description” would be easier to understand.